integration/kubernetes/conf/worker.properties.template

```diff
@@ -0,0 +1,14 @@
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+ALLUXIO_WORKER_MEMORY_SIZE=768MB
+ALLUXIO_WORKER_PORT=29998
+ALLUXIO_WORKER_DATA_PORT=29999
+ALLUXIO_WORKER_WEB_PORT=30001
```

Comment:
is 30000 already taken by something else in kubernetes?

Reply:
yes, the k8s dashboard listens on 30000